lib/jsdom/living/nodes/HTMLFormControlsCollection-impl.js

```diff
@@ -0,0 +1,30 @@
+"use strict";
+
+const HTMLCollectionImpl = require("./HTMLCollection-impl").implementation;
+const RadioNodeList = require("../generated/RadioNodeList");
+
+exports.implementation = class HTMLFormControlsCollectionImpl extends HTMLCollectionImpl {
+  namedItem(key) {
+    if (!key) {
+      return null;
+    }
+    this._update();
+    const nodeList = RadioNodeList.createImpl(this._globalObject, [], {
+      element: this,
+      query: () => this._list.reduce((a, b) => {
+        if (b.getAttributeNS && (b.getAttributeNS(null, "id") === key || b.getAttributeNS(null, "name") === key)) {
```

Comment:
When would getAttributeNS not be truthy?

Reply:
just looking at the code in HTMLCollection-impl.js I didn't see any obvious check and wanted to play it safe... but yeah I guess if getAttributeNS is missing here there must a bigger problem somewhere else so this is not the place for that check...